data/json/overmap_terrain_military.json

```diff
@@ -100,6 +109,15 @@
     "see_cost": 2,
     "flags": [ "KNOWN_UP", "NO_ROTATE" ]
   },
+  {
+    "type": "overmap_terrain",
+    "id": "lmoe_under_empty",
+    "name": "LMOE shelter",
+    "sym": 43,
```

Comment:
```suggestion
    "sym": "+",
```

Reply:
Is that what Travis didn't like?

The error message seems quite vague if that's the case.